### PR TITLE
my expert page to bs4

### PR DESCRIPTION
**Pull request by kristinmerbach**
# PR Checklist

Please check if your PR fulfills the following requirements
- [ ] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [ ] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [ ] Any endpoint modified in the PR only responds to the expected MIME types.
- [ ] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [ ] There are no inline styles added
- [ ] There are no inline javascript added
- [ ] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [ ] Code does not use .html_safe
- [ ] All images added/updated have alt text
- [ ] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket:  https://www.pivotaltracker.com/story/show/187379749

# A brief description of the changes

Current behavior:

New behavior:

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/views/insured/families/brokers.html.erb [hunk 1]
```diff
@@ -1,27 +1,48 @@
...
+    <h1><%= l10n('my_broker') %></h1>
+    <section class="rounded-lg bg-white border p-3 my-4 col-9">
+      <% if @person.has_active_employee_role? && has_writing_agent?(@employee_role) %>
+        <%= render 'employee_broker', title: nil %>
+      <% elsif @person.has_consumer_role? && has_writing_agent?(@person)%>
+        <%= render 'consumer_broker', title: nil %>
+      <% end %>
+    </section>
+  <% end %>
+  <a href="<%= main_app.family_account_path(tab: 'home') %>" class="button mt-4">Return Home</a>
```
Comment (charlienparker) [inline app/views/insured/families/brokers.html.erb:19]: localize Return Home

**Review by charlienparker — APPROVED** [review]
small l10n but otherwise looks good

File: app/views/insured/families/_consumer_broker.html.erb [hunk 2]
```diff
@@ -1,3 +1,62 @@
...
+      </svg>
+      <h2 class="ml-2"><%= writing_agent&.person&.first_name&.capitalize %> <%= writing_agent&.person&.last_name&.capitalize %></h2>
+    </div>
+    <a href="#destroyBroker" data-toggle="modal" data-target="#destroyBroker" class="button outline error">Remove Expert</a>
```
Comment (ATBull81) [inline app/views/insured/families/_consumer_broker.html.erb:11]: quick l10n

**Review by ATBull81 — APPROVED** [review]
Looks good other than two l10ns

**Pull request merged**